src/redisenterprise/azext_redisenterprise/azext_metadata.json

```diff
@@ -1,3 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.15.0"
+    "azext.isPreview": true,
```

Comment:
It seems the previous version is not preview. You should remove this line.

Reply:
removed